packages/browser/src/node/pool.ts

```diff
@@ -156,6 +161,11 @@ export function createBrowserPool(ctx: Vitest): ProcessPool {
     async close() {
       await Promise.all([...providers].map(provider => provider.close()))
       providers.clear()
+      ctx.resolvedProjects.forEach((project) => {
+        project.browser?.state.orchestrators.forEach((orchestrator) => {
+          orchestrator.$close()
```

Comment:
Nice, so the `$close` is synchronous method that immediatelly closes everything?